packages/react/src/components/ErrorSummary/ErrorSummaryList.tsx

```diff
@@ -1,9 +1,11 @@
-import { List } from '../List';
+import { forwardRef } from 'react';
+import { List, type ListUnorderedProps } from '../List';
 
-export type ErrorSummaryListProps = React.ComponentProps<typeof List.Unordered>;
+export type ErrorSummaryListProps = Omit<ListUnorderedProps, 'ref'>;
 
-export default function ErrorSummaryList({ ...rest }: ErrorSummaryListProps) {
-  return <List.Unordered {...rest} />;
-}
-
-ErrorSummaryList.displayName = 'ErrorSummaryList';
+export const ErrorSummaryList = forwardRef<
+  HTMLOListElement,
+  ErrorSummaryListProps
+>(function ErrorSummaryList({ ...rest }: ErrorSummaryListProps, ref) {
+  return <List.Unordered {...rest} ref={ref} />;
+});
```

Comment:
Since you removed the list root, setting size on `ErrorSummary` does no longer change the size

Reply:
I think we should discuss ErrorSummary sizing also post our conversations on subcomponents vs using the original components directly, but I'll add a fix for scaling ErrorSummary here for now :)